policyengine_canada/variables/gov/provinces/on/tax/income/credits/ostc/household_members.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class household_members(Variable):
```

Comment:
For consistency with US variable names. Please also move this to `variables/household/household_size.py`
```suggestion
class household_size(Variable):
```